tspi/tpm.go

```diff
@@ -147,6 +147,22 @@ func (tpm *TPM) ExtendPCR(pcr int, data []byte, eventtype int, event []byte) err
 	return err
 }
 
+// GetPCRValues obtains the PCR values from the TPM.
+func (tpm *TPM) GetPCRValues() ([][]byte, error) {
```

Comment:
Aren't PCR values a well known length? Couldn't this signature be:

```go
func (tpm *TPM) GetPCRValues() ([24][sha1.Size]byte, error) {
```

Reply:
It could be but then we wouldn't be able to return nil on error.
Not sure how important that is though

Reply:
That's fine. The error already indicates that the other values are invalid.

Reply:
per out of band conversation, let's just go with [][]byte